libbeat/outputs/outputs.go

```diff
@@ -6,11 +6,11 @@ import (
 )
 
 type MothershipConfig struct {
-	Save_topology     bool
+	SaveTopology      bool `yaml:"save_topology"`
 	Host              string
 	Port              int
 	Hosts             []string
-	LoadBalance       *bool
+	LoadBalance       *bool `yaml:"loadbalance"`
```

Comment:
@urso I added this change as I expected it to be called load_balance in the config, but it is loadbalance ...